[2025-05-17T02:37:08.432+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: uhi_index_analytics_pipeline_decompose.convert_zoning_to_tiff manual__2025-05-13T13:33:48.016143+00:00 [queued]>
[2025-05-17T02:37:08.484+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: uhi_index_analytics_pipeline_decompose.convert_zoning_to_tiff manual__2025-05-13T13:33:48.016143+00:00 [queued]>
[2025-05-17T02:37:08.486+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-05-17T02:37:08.489+0000] {taskinstance.py:1280} INFO - Starting attempt 3 of 3
[2025-05-17T02:37:08.492+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-05-17T02:37:08.542+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): convert_zoning_to_tiff> on 2025-05-13 13:33:48.016143+00:00
[2025-05-17T02:37:08.567+0000] {standard_task_runner.py:55} INFO - Started process 72 to run task
[2025-05-17T02:37:08.574+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'uhi_index_analytics_pipeline_decompose', 'convert_zoning_to_tiff', 'manual__2025-05-13T13:33:48.016143+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/uhi_pipeline_dag_decompose.py', '--cfg-path', '/tmp/tmpuydlua5a']
[2025-05-17T02:37:08.576+0000] {standard_task_runner.py:83} INFO - Job 177: Subtask convert_zoning_to_tiff
[2025-05-17T02:37:08.582+0000] {taskinstance.py:1077} INFO - Dependencies not met for <TaskInstance: uhi_index_analytics_pipeline_decompose.convert_zoning_to_tiff manual__2025-05-13T13:33:48.016143+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2025-05-17T02:37:08.584+0000] {taskinstance.py:1077} INFO - Dependencies not met for <TaskInstance: uhi_index_analytics_pipeline_decompose.convert_zoning_to_tiff manual__2025-05-13T13:33:48.016143+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2025-05-17T02:37:08.595+0000] {local_task_job.py:147} INFO - Task is not able to be run
[2025-05-17T02:37:08.807+0000] {task_command.py:388} INFO - Running <TaskInstance: uhi_index_analytics_pipeline_decompose.convert_zoning_to_tiff manual__2025-05-13T13:33:48.016143+00:00 [running]> on host 869b990513f1
[2025-05-17T02:37:09.058+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uhi_index_analytics_pipeline_decompose
AIRFLOW_CTX_TASK_ID=convert_zoning_to_tiff
AIRFLOW_CTX_EXECUTION_DATE=2025-05-13T13:33:48.016143+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-05-13T13:33:48.016143+00:00
[2025-05-17T02:37:11.796+0000] {logging_mixin.py:137} INFO - ['C1-2', 'C1-3', 'C1-4', 'C1-5', 'C2-1', 'C2-2', 'C2-3', 'C2-4', 'C2-5']
[2025-05-17T02:37:11.806+0000] {logging_mixin.py:137} INFO - ['C1', 'C2']
[2025-05-17T02:37:10.937+0000] {logging_mixin.py:137} WARNING -   0%|          | 0/558 [00:00<?, ?it/s]
[2025-05-17T02:37:13.410+0000] {logging_mixin.py:137} WARNING -   0%|          | 1/558 [00:02<22:57,  2.47s/it]
[2025-05-17T02:37:14.813+0000] {logging_mixin.py:137} WARNING -   0%|          | 2/558 [00:03<17:04,  1.84s/it]
[2025-05-17T02:37:16.454+0000] {logging_mixin.py:137} WARNING -   1%|          | 3/558 [00:05<16:11,  1.75s/it]
[2025-05-17T02:37:17.705+0000] {logging_mixin.py:137} WARNING -   1%|          | 4/558 [00:06<14:20,  1.55s/it]
[2025-05-17T02:37:19.082+0000] {logging_mixin.py:137} WARNING -   1%|          | 5/558 [00:08<13:44,  1.49s/it]
[2025-05-17T02:37:20.127+0000] {logging_mixin.py:137} WARNING -   1%|1         | 6/558 [00:09<12:18,  1.34s/it]
[2025-05-17T02:37:21.296+0000] {logging_mixin.py:137} WARNING -   1%|1         | 7/558 [00:10<11:46,  1.28s/it]
[2025-05-17T02:37:22.365+0000] {logging_mixin.py:137} WARNING -   1%|1         | 8/558 [00:11<11:08,  1.22s/it]
[2025-05-17T02:37:23.793+0000] {logging_mixin.py:137} WARNING -   2%|1         | 9/558 [00:12<11:43,  1.28s/it]
[2025-05-17T02:37:24.662+0000] {logging_mixin.py:137} WARNING -   2%|1         | 10/558 [00:13<10:32,  1.15s/it]
[2025-05-17T02:37:25.377+0000] {logging_mixin.py:137} WARNING -   2%|1         | 11/558 [00:14<09:17,  1.02s/it]
[2025-05-17T02:37:26.508+0000] {logging_mixin.py:137} WARNING -   2%|2         | 12/558 [00:15<09:35,  1.05s/it]
[2025-05-17T02:37:27.325+0000] {logging_mixin.py:137} WARNING -   2%|2         | 13/558 [00:16<08:55,  1.02it/s]
[2025-05-17T02:37:28.081+0000] {logging_mixin.py:137} WARNING -   3%|2         | 14/558 [00:17<08:17,  1.09it/s]
[2025-05-17T02:37:28.904+0000] {logging_mixin.py:137} WARNING -   3%|2         | 15/558 [00:17<08:01,  1.13it/s]
[2025-05-17T02:37:29.651+0000] {logging_mixin.py:137} WARNING -   3%|2         | 16/558 [00:18<07:37,  1.18it/s]
[2025-05-17T02:37:30.523+0000] {logging_mixin.py:137} WARNING -   3%|3         | 17/558 [00:19<07:41,  1.17it/s]
[2025-05-17T02:37:31.615+0000] {logging_mixin.py:137} WARNING -   3%|3         | 18/558 [00:20<08:19,  1.08it/s]
[2025-05-17T02:37:33.206+0000] {logging_mixin.py:137} WARNING -   3%|3         | 19/558 [00:22<10:06,  1.12s/it]
[2025-05-17T02:37:34.718+0000] {logging_mixin.py:137} WARNING -   4%|3         | 20/558 [00:23<11:07,  1.24s/it]
[2025-05-17T02:37:35.728+0000] {logging_mixin.py:137} WARNING -   4%|3         | 21/558 [00:24<10:29,  1.17s/it]
[2025-05-17T02:37:36.474+0000] {logging_mixin.py:137} WARNING -   4%|3         | 22/558 [00:25<09:19,  1.04s/it]
[2025-05-17T02:37:37.195+0000] {logging_mixin.py:137} WARNING -   4%|4         | 23/558 [00:26<08:26,  1.06it/s]
[2025-05-17T02:37:37.875+0000] {logging_mixin.py:137} WARNING -   4%|4         | 24/558 [00:26<07:42,  1.15it/s]
[2025-05-17T02:37:38.591+0000] {logging_mixin.py:137} WARNING -   4%|4         | 25/558 [00:27<07:17,  1.22it/s]
[2025-05-17T02:37:39.322+0000] {logging_mixin.py:137} WARNING -   5%|4         | 26/558 [00:28<07:02,  1.26it/s]
[2025-05-17T02:37:40.156+0000] {logging_mixin.py:137} WARNING -   5%|4         | 27/558 [00:29<07:08,  1.24it/s]
[2025-05-17T02:37:40.730+0000] {logging_mixin.py:137} WARNING -   5%|5         | 30/558 [00:29<04:03,  2.17it/s]
[2025-05-17T02:37:41.423+0000] {logging_mixin.py:137} WARNING -   6%|5         | 31/558 [00:30<04:28,  1.96it/s]
[2025-05-17T02:37:42.418+0000] {logging_mixin.py:137} WARNING -   6%|5         | 32/558 [00:31<05:27,  1.61it/s]
[2025-05-17T02:37:43.216+0000] {logging_mixin.py:137} WARNING -   6%|5         | 33/558 [00:32<05:49,  1.50it/s]
[2025-05-17T02:37:44.137+0000] {logging_mixin.py:137} WARNING -   6%|6         | 34/558 [00:33<06:24,  1.36it/s]
[2025-05-17T02:37:45.455+0000] {logging_mixin.py:137} WARNING -   6%|6         | 35/558 [00:34<07:46,  1.12it/s]
[2025-05-17T02:37:46.862+0000] {logging_mixin.py:137} WARNING -   6%|6         | 36/558 [00:35<09:01,  1.04s/it]
[2025-05-17T02:37:47.680+0000] {logging_mixin.py:137} WARNING -   7%|6         | 37/558 [00:36<08:27,  1.03it/s]
[2025-05-17T02:37:48.878+0000] {logging_mixin.py:137} WARNING -   7%|6         | 38/558 [00:37<09:00,  1.04s/it]
[2025-05-17T02:37:50.401+0000] {logging_mixin.py:137} WARNING -   7%|6         | 39/558 [00:39<10:12,  1.18s/it]
[2025-05-17T02:37:51.740+0000] {logging_mixin.py:137} WARNING -   7%|7         | 40/558 [00:40<10:35,  1.23s/it]
[2025-05-17T02:37:53.509+0000] {logging_mixin.py:137} WARNING -   7%|7         | 41/558 [00:42<11:57,  1.39s/it]
[2025-05-17T02:37:53.728+0000] {local_task_job.py:112} ERROR - Received SIGTERM. Terminating subprocesses
[2025-05-17T02:37:53.735+0000] {process_utils.py:133} INFO - Sending Signals.SIGTERM to group 72. PIDs of all processes in the group: [72]
[2025-05-17T02:37:53.740+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 72
[2025-05-17T02:37:53.746+0000] {taskinstance.py:1479} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-05-17T02:37:53.749+0000] {logging_mixin.py:137} WARNING -   7%|7         | 41/558 [00:42<08:59,  1.04s/it]
[2025-05-17T02:37:53.805+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/uhi_pipeline_dag_decompose.py", line 178, in convert_zoning_to_tiff
    tiff_transform.input_data_handle("nyco.geojson", type="nyco", resolution=30)
  File "/opt/airflow/project_code/data_pipeline_test/tiff_transform.py", line 57, in input_data_handle
    zoning_nyco_tiff(read_path, save_path, resolution)
  File "/opt/airflow/project_code/data_pipeline_test/tiff_transform.py", line 413, in zoning_nyco_tiff
    overlay_level2, _ = rasterize(df, geometry_col, "overlay_level2", ["mode"], gt, height, width)
  File "/opt/airflow/project_code/data_pipeline_test/tiff_transform.py", line 255, in rasterize
    clipped_df = df.iloc[intersect_idx]
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/indexing.py", line 931, in __getitem__
    return self._getitem_axis(maybe_callable, axis=axis)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/indexing.py", line 1557, in _getitem_axis
    return self._get_list_axis(key, axis=axis)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/indexing.py", line 1530, in _get_list_axis
    return self.obj._take_with_is_copy(key, axis=axis)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/generic.py", line 3628, in _take_with_is_copy
    result = self.take(indices=indices, axis=axis)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/generic.py", line 3616, in take
    indices, axis=self._get_block_manager_axis(axis), verify=True
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/internals/managers.py", line 864, in take
    new_labels = self.axes[axis].take(indexer)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/indexes/range.py", line 438, in take
    return self._int64index.take(
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1481, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2025-05-17T02:37:54.006+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=uhi_index_analytics_pipeline_decompose, task_id=convert_zoning_to_tiff, execution_date=20250513T133348, start_date=20250517T023708, end_date=20250517T023753
[2025-05-17T02:37:54.113+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 177 for task convert_zoning_to_tiff (Task received SIGTERM signal; 72)
[2025-05-17T02:37:54.230+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=72, status='terminated', exitcode=1, started='02:37:07') (72) terminated with exit code 1
[2025-05-17T02:37:54.232+0000] {local_task_job.py:208} INFO - Task exited with return code 143
[2025-05-17T02:37:54.324+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
